.build.ps1

```diff
@@ -72,3 +72,6 @@ ktbacr.azurecr.io/trivy/trivy-java-db:1
 oras pull /trivy/trivy-config-checks-db: -o <output_path>
 trivy config --config-checks-db <output_path> <target_directory>
 
+# TODO: add image scanning example
```

Comment:
suggestion to use examples such as:

`trivy image --vuln-type os,library --ignore-unfixed --severity HIGH,CRITICAL --ignore-policy .trivyignore myapp:latest`

`trivy image --ignore-unfixed node:18`

`trivy image --severity HIGH,CRITICAL ubuntu:20.04`

Reply:
Added examples